Mathlib/Order/GameAdd.lean

```diff
@@ -4,10 +4,11 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Junyan Xu
 
 ! This file was ported from Lean 3 source module order.game_add
-! leanprover-community/mathlib commit 99e8971dc62f1f7ecf693d75e75fbbabd55849de
+! leanprover-community/mathlib commit fee218fb033b2fd390c447f8be27754bc9093be9
 ! Please do not edit these lines, except to modify the commit id
 ! if you have ported upstream changes.
 -/
+import Mathlib.Data.Sym.Sym2
 import Mathlib.Logic.Relation
 import Mathlib.Order.Basic
```

Comment:
Mathport output deletes this line, see https://leanprover-community.github.io/mathlib-port-status/file/order/game_add?range=99e8971dc62f1f7ecf693d75e75fbbabd55849de..fee218fb033b2fd390c447f8be27754bc9093be9